src/directives/routeHref.js

```diff
@@ -8,13 +8,17 @@ function routeHrefFactory (Route, $location, $timeout) {
     if (iAttrs.ignoreHref === undefined) {
       iElement.click((event) => {
         event.preventDefault();
-        let url = iElement.attr('href');
+        let urlPath = iElement.attr('href');
 
-        if (!Route.isHtml5ModeEnabled()) {
-          url = url.replace(/^#/, '');
+        if (event.metaKey) {
```

Comment:
has this logic been tested on a box that is not localhost and uses https ? wondering 

a) how that split is not tripping up on the http:// (to me it implies that the http(s) is not present)
b) whether we are dropping the https because location assumes http ?

Reply:
also make sure we test with a windows browser as that is the target. If it works on OSX that is a bonus

Reply:
Yes looks like a bug good spot. I've fixed it and now working on the test for it.

Reply:
event.metaKey won't work on windows - I believe you'll need event.ctrlKey for that. However...

The only reason the route-href needs to do anything besides add the href attribute, is to support HTML 5 mode. Without some special handling, in HTML 5 mode, the browser will do full page reloads, which isn't what you want in a SPA.

My recommendation is still as per the JIRA ticket - check if HTML 5 mode is enabled, if it is not (or "ignoreHref" is on the element - i.e. retain existing behaviour), don't even add the click handler, just short circuit right there. That's should be all we need for now.

As also mentioned in the ticket, when or if we eventually want to use HTML 5 mode, *then* it'll be worthwhile doing it properly - which entails supporting middle click, meta click (mac), ctrl click (windows / linux) and a more thorough investigation.

Reply:
Ok mate no worries, doing it this way sounds ALOT simpler.